dynamic/conditions/conditions.go

```diff
@@ -140,6 +140,29 @@ func (do *DynamicOptions) ReadConditions() (*unstructured.Unstructured, []kmapi.
 	return resp, conditions, err
 }
 
+func (do *DynamicOptions) UpdateConditions(conditions []kmapi.Condition) error {
+	obj, err := do.Client.Resource(do.GVR).Namespace(do.Namespace).Get(context.TODO(), do.Name, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	unstrConds := make([]interface{}, len(conditions))
+	for i := range conditions {
+		cond, err := runtime.DefaultUnstructuredConverter.ToUnstructured(&conditions[i])
+		if err != nil {
+			return err
+		}
+		unstrConds[i] = cond
```

Comment:
`unstrConds[i] = conditions[i]`

I think that should be enough.

Reply:
It gets paniced! 
> Observed a panic: &errors.errorString{s:"cannot deep copy v1.Condition"} (cannot deep copy v1.Condition)